1.Practicals_C/Miscc/revnum.c

```diff
@@ -0,0 +1,21 @@
+// Program to find reverse of a five digit number.
+#include <stdio.h>
+int main()
+{
+    int n, d5, d4, d3, d2, d1;
+    long int revnum;
+    printf("\nEnter a five digit number:");
+    scanf("%d", &n);
+    d5 = n % 10;
+    n = n / 10;
+    d4 = n % 10;
+    n = n / 10;
```

Comment:
Use loop to implement the same. That would be the better way.

Reply:
> Use loop to implement the same. That would be the better way.

Sure Sir, I will try to do that. Thank you for your suggestion.